composer.json

```diff
@@ -35,7 +35,8 @@
         "symfony/serializer": "^3.4|^4",
         "symfony/yaml": "^3.4|^4",
         "friendsofphp/php-cs-fixer": "^2.2",
-        "symfony/web-profiler-bundle": "^3.4|^4.0"
+        "symfony/web-profiler-bundle": "^3.4|^4.0",
+        "phpdocumentor/type-resolver": "~0.2.1"
```

Comment:
We require at least 0.2.1 version of type-resolver, because of following bug https://github.com/phpDocumentor/TypeResolver/commit/                               e224fb2ea2fba6d3ad6fdaef91cd09a172155ccb

Reply:
Removed. Probably, it is not required after https://github.com/FriendsOfSymfony/FOSElasticaBundle/commit/434d19c15a90ce8c2e0dace98b98a56dc38f631e#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780